src/_pytest/config/__init__.py

```diff
@@ -40,6 +40,7 @@
 import _pytest._code
 import _pytest.deprecated
 import _pytest.hookspec
+from .. import deprecated
```

Comment:
This file already does `import _pytest.deprecated` above, maybe better to reuse that instead of reimporting it with a different name?

Reply:
Good catch, thanks!